sdk/ai/ai-inference-rest/src/parameters.ts

```diff
@@ -48,3 +52,55 @@ export type GetChatCompletionsParameters = GetChatCompletionsHeaderParam &
   GetChatCompletionsBodyParam &
   RequestParameters;
 export type GetModelInfoParameters = RequestParameters;
+
+export interface GetEmbeddingsHeaders {
+  /**
+   * Controls what happens if extra parameters, undefined by the REST API,
+   * are passed in the JSON request payload.
+   * This sets the HTTP request header `extra-parameters`.
+   */
+  "extra-parameters"?: ExtraParameters;
+}
+
+export interface GetEmbeddingsBodyParam {
+  body?: {
+    input: string[];
+    dimensions?: number;
+    encoding_format?: EmbeddingEncodingFormat;
+    input_type?: EmbeddingInputType;
+  };
```

Comment:
please add "model?: string" to text and image embeddings, similar to chat completions. It's required for GH endpoint.

Reply:
[Updated](https://github.com/Azure/azure-sdk-for-js/pull/30379/commits/74fe10df32a86bd41c499f0e84ba2363df5ee59b)